README.md

```diff
@@ -76,6 +76,7 @@ Want to add something? Open a PR :)
 - [doppins](https://doppins.com)
 - [tidelift](https://tidelift.com)
 - [fossa](https://fossa.com)
+- [diun](https://crazymax.dev/diun/)
```

Comment:
Mind linking to the github [repo](https://github.com/crazy-max/diun) of the project instead of the dev's page? I'll approve the PR right after - thanks!